kalite/testing/base_environment.py

```diff
@@ -245,3 +254,42 @@ def do_fake_registration():
     device = Device.get_own_device()
     device_zone = DeviceZone(device=device, zone=zone)
     device_zone.save()
+
+
+def _make_video(context):
+    lang_code = "en"
+    youtube_id = "my_cool_id"
+    item_dict = {
+        "title": "Subtitled Video",
+        "description": "A video with subtitles",
+        "available": True,
+        "kind": "Video",
+        "id": "video_with_subtitles",
+        "slug": "video_with_subtitles",
+        "path": "khan/video_with_subtitles",
```

Comment:
So follow up -- is there any expectation that such an Item will exist on the test server? Or would it only exist if a content pack is installed?